apps/user-profile/src/app/pages/profile-page/profile-page.component.html

```diff
@@ -11,7 +11,7 @@
         <div [matTooltip]="translateService.currentLang === 'en' ? attribute.tooltip_en : attribute.tooltip_cz"
              matTooltipPosition="above">{{translateService.currentLang === 'en' ? attribute.displayName_en : attribute.displayName_cz}}</div>
       </td>
-      <td [ngSwitch]="attribute.attribute.friendlyName">
+      <td [ngSwitch]="attribute.attribute.friendlyName" class="w-75">
```

Comment:
This is not sufficient to fix the issue:
![image](https://user-images.githubusercontent.com/26108901/113109477-534dd180-9206-11eb-80b3-96b33fb80d38.png)

The problem is, that the width of the whole table is not set to the whole available space.